doc/proposals/2019_generated promql parser.md

```diff
@@ -5,39 +5,43 @@ menu: proposals
+It seems that none of the active members of the Prometheus project has a deep understanding of the parser code and is willing to maintain it or review complex PRs on it (based on the discussion in <https://github.com/prometheus/prometheus/pull/6061>).
 
-Also there seem to be some hidden features and edge cases that nobody is aware of (See <https://github.com/prometheus/prometheus/pull/6198> and <https://github.com/prometheus/prometheus/issues/6065>).
+There also seem to be some hidden features and edge cases that nobody is aware of (see <https://github.com/prometheus/prometheus/pull/6198> and <https://github.com/prometheus/prometheus/issues/6065>).
 
-The code seems to be far more complex than necessary and before continuing feature development on it, major refactoring would be needed.
+The code seems to be far more complex than necessary and would require significant refactoring before continuing to develop new features on it.
 
-### Using the parser for the upcoming language server 
+### Using the Parser for the Upcoming Language Server 
 
-For the upcoming PromQL language Server (see [proposal](https://github.com/slrtbtfs/promql-lsp/blob/master/doc/proposals/2019_promql-language-server.md)) it is necessary to analyze PromQL queries.
+The upcoming PromQL language Server (see [proposal](https://github.com/slrtbtfs/promql-lsp/blob/master/doc/proposals/2019_promql-language-server.md)) requires a parser in order to analyze PromQL queries.
 
-For this it is intended to use the same PromQL parser as prometheus. To enable that use case, changes to the parser have been [proposed](https://github.com/slrtbtfs/promql-lsp/blob/master/doc/proposals/2019_promql-parser-improvements.md). These mainly concern the _output_ of the parser and have already been partly implemented.
+For purposes of compatibility, maintainability, and code reuse, it is intended for the language server to use the same PromQL parse as Prometheus.
+To enable that use case, some enhancements to the existing parser have been [proposed](https://github.com/slrtbtfs/promql-lsp/blob/master/doc/proposals/2019_promql-parser-improvements.md).
+These mainly concern the _output_ of the parser and have already been partly implemented.
+However, some of the features described there, such as error recovery, would be difficult to implement in the existing parser.
 
-However some of the features described there, like error recovery, would be quite hard to implement with the current parser.
+#### Producing a Deeper Syntax Tree
 
-#### Producing a deeper syntax tree
-
-Consider the query:
+Consider the following PromQL query:
 
     metric{label=="value", foo~="ba."}[5m] offset 3h
 
-This is how the corresponding syntax tree looks like with the current parser:
+The cooresponding syntax tree produced by the current parser looks something like the following:
```

Comment:
There seems to be one o too much.

Reply:
Apparently the correct spelling is corresponding  https://www.dict.cc/englisch-deutsch/corresponding.html

Reply:
:tada:

Reply:
Apparently the correct spelling is "corresponding" (with two 'r's) https://www.dict.cc/englisch-deutsch/corresponding.html

Reply:
found some more typos too :p

Reply:
The above comment wasn't to be sent twice. Github just didn't show it on the first attempt, that's why I sent it again.